src/Robots.php

```diff
@@ -140,7 +140,7 @@ protected function disallowedUrls()
     {
         // If not public, disallow all
         if (!$this->isPublic()) {
-            return array("/");
+            return ["/"];
```

Comment:
I think the IDE changed this on accident.

Reply:
All good bro